### DISCUSSION TITLE
We need old feed back

### DISCUSSION THREAD

**Discussion by linuxmobile** [community/community#66244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

You committed an aberration by removing the previous feed.
In the current feed, no matter how many filters you have added, I can't find the information I'm interested in. Here is an example:

![image](https://github.com/community/community/assets/10554636/7adc2830-ca6e-43bb-b813-98e0efdac44f)
![image](https://github.com/community/community/assets/10554636/723294e1-1c86-4f9d-b624-033730c5c8c4)

The previous feed was much, much better. Please, stop touching things that are not extremely necessary. Don't the Devs that are taking care of these tasks use github?

**Comment by liuyib** [comment]
+1, I agree with you. The new homepage is completely useless, thanks GitHub. :(

**Reply by esimov** [reply]
The Github UI and UX by each upgrade is getting worser and worser. The github feed slowly became a total mess and uselessness. With the new feed they achieved that my feed looks like this: 

![image](https://github.com/community/community/assets/883386/2179928d-f678-4736-ad89-29aacdfd2f41)

The most important part for a developer as a repo owner has jus been completely ridded off: who starred the project or the new followers are just not important anymore. I don't really get what drives the Github executives or whoever for making these decisions. A total useless crap.

**Reply by liuyib** [reply]
https://github.com/Gerrit0/old-github-feed
Using this script you can restore the old version of the feed.

**Reply by esimov** [reply]
No, I don't want, I want GH to bring back the old feed.

**Reply by bytefish** [reply]
I don’t know if it has been mentioned elsewhere. The original feed is still available at: https://github.com/dashboard-feed.

**Reply by michalrana** [reply]
Am also having the same issue and I don't find any solution[.]() Is there anyone who can give me the solution? I think we all must google this issue for quick solution.

Moderator Edit: Removing "hidden" link

**Comment by mjpitz** [comment]
A big issue I have with the new feed is that mine is effectively filled now with a bunch of junk. It's why I never voluntarily switched over. I've had my GitHub for just over a decade now and I'm getting useless updates about projects / repositories I starred ages ago, that I have no interest in today. I get I can go in and unstar things, but that's a whole lot of work to do. As a result of this change over, the things I actually cared about (what folks I was following were looking at) is now lost in all the noise.

**Reply by linuxmobile** [reply]
same :(

**Reply by gregdeane** [reply]
Total junk. Now I can't find any of the things that people I follow add stars to. Totally worthless now.

**Reply by MarkoRamius** [reply]
_is effectively filled now with a bunch of junk_

One wonders, whether that's Microsoft at work.
When I look at a vanilla Windows installation, that's what I see first: _is effectively filled now with a bunch of junk_

**Comment by jh0072110** [comment]
what the hell of new homepage! i can't find anything i want

**Comment by HCLarsen** [comment]
Agreed. The new feed is terrible. 

First that no one seems to have wanted the For You feed, and now it's being forced on us by combining it with the Following Feed. 

Second, the design of the feed is bad. We were all familiar with the old one, so why change it?

It seems like Microsoft is really obsessed with creating terrible software.

**Reply by takeda** [reply]
It feels like steps toward "pay to make your repo updates more visible".

**Reply by aboood40091** [reply]
It's not just Microsoft, but Google too. Go see how Google Images reverse search is progressively becoming WORSE. It's extremely frustrating.

It was the same situation as the For you feed on GitHub. First, we had the normal Google images reverse search, which was ALREADY perfectly fine. Then, Google introduced Google Lens, as an alternative at first; Google Lens would be the default you got but you had an option that takes you back to the old one (just like how the For you feed was separate on GitHub). But then a few months later Google Lens became enforced. You CANNOT go back to the old Google Images reverse search anymore. They literally had a button that takes you back, but no, had to nuke that button because OBVIOUSLY less features = improvement nowadays (the exact same button is still there, it just doesn't take you back anymore). I don't even understand the logic behind this. And then when people complain about it on the Google Support forums, they get the infuriating reply "Dev team is trying its best to deliver to you the best service blah blah blah we hope it improves in the future" all while Google services become worse. The nerve of these people...

I know this is irrelevant to the current issue, but it's so frustrating that I needed to vent somewhere. 🥲

**Comment by sgf** [comment]
The homepage of the new version is very confusing, and I can't see which valuable repositories have been starred by the people I follow.
I want to go back to the old home page.

**Reply by linuxmobile** [reply]
Same :(

**Reply by gregdeane** [reply]
Same 😡

**Reply by yisier** [reply]
Same

**Comment by sgf** [comment]
I don't know why in the programming or software industry.
I also found that many unprofessional people occupy highly professional positions.
In the C# community, Microsoft let a person who has no trace of writing code on the github page participate in managing the direction of the C# language.
In Github, it seems that another product manager who does not use github is found to design the homepage of github.
Why do these absurd things always happen in Microsoft's business lines?

**Reply by liuyib** [reply]
maybe KPI   :)

**Reply by linuxmobile** [reply]
It is really confusing where they are aiming the Github frontend at

**Comment by xingkong123600** [comment]
just useless!! I can't see which valuable repositories have been starred by the people I follow!!

**Reply by linuxmobile** [reply]
Same problem :(

**Comment by zach-betz-hln** [comment]
Same here. I only care about the commits and PRs of people that I follow... and now I can't see that info.

**Comment by u3u** [comment]
https://github.com/dashboard-feed

**Reply by linuxmobile** [reply]
Yes, I saw it. But have you seen what it looks like on a 1440 monitor? It looks horrible.

**Reply by u3u** [reply]
I recovered through it!
https://github.com/Gerrit0/old-github-feed

**Reply by linuxmobile** [reply]
![image](https://github.com/community/community/assets/10554636/1ddf9f02-31f1-4f27-93e1-2df7c420c2ec)

By pasting the code into the DevTools console, it works.

**Comment by takeda** [comment]
You added the absolutely useless social part to github, and now you're removing the actually useful feed? WTF? What next? Political news section?

This trend makes me think of just switching to gitlab.

**Reply by esimov** [reply]
I have to completely agree with you. From the Microsoft acquisition Gitlab slowly became a social platform and with each new upgrade it's getting worse and worse. I was at the edge of switching to Gitlab when that happened, but I tempered myself. The thing is since Microsoft, the Github UI/UX experience is a bunch of a mess and useless junk.

**Reply by quantuminformation** [reply]
soon you can add your pronouns to your js variables

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr><td width="50%" align="center"><b>Before (the shit💩 new feed)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066442147.jpg"/>
        </p>
     </td>
  </tr>
  <tr><td width="50%" align="center"><b>After use (the old one)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066437621.jpg"/>
        </p>
     </td>
  </tr>
</table>

**Comment by hnhparitosh** [comment]
At least bring back the option to toggle between the old feed and 'For you'.

**Comment by HinTak** [comment]
Really shit and waste of screen estate. I don't need to watch people thumb up this or that!

**Comment by cyring** [comment]
Plz bring back old feedback: I miss my repo stars activities.

**Comment by Jappie3** [comment]
+1, the most valuable information is just gone with no way to bring it back, thanks microsoft.
Github is not social media, stop trying to use fancy algorithms to show me 'stuff that I want'. I don't care about updates for a random project I starred 2 years ago, if I want to know about it I'll go look for it.
A plain old list of projects starred by people I follow & MRs was fine.

**Reply by smxi** [reply]
I have no idea who decided this idiotic new feed with no way to switch back to the fairly useful and simple old one was an idea worth implementing, but they clearly have no business being involved in these types of decisions.

My new 'feed' is totally useless, and seems stuck on one ancient starred project that did a release. 

I worry about the radical incompetence level on display here, but then again, using github was always something I did to help my users find the project and file issues, I have always disliked github, and was expecting some total garbage to be released after microsoft bought github. 
This failed idea certainly seems to confirm my expectations.

At least give users the choice, you can just make it a toggle selector, wording something like this:

original, usable, useful feed method | new, totally useless and failed feed method. 

But really, who made  this decision, was it a committee? It clearly was not made by an actual active user of this service, with active repos that they run, which is of  course what I want to see on my feed primarily, and now there is zero, I have literally no idea what is going on with my projects now anymore.

Just fire whoever did this, consider it their resignation, and restore the old one, and find something useful to work on.

The link to old feed isn't useful because there's nothing else on that page except the feed, so clearly all they have to do is add in the selector for which feed to see, then here's a radical suggestion, watch and see how many people select either. If the number for the old is greater than a few percent, then leave the option and find other ways to break things that worked fine.

I agree, the confusion involved in trying to make github a social media platform is massive, this is a code repository. I really wish googlecode had not fallen by the wayside, that was a far superior code repo in almost every way.

**Comment by Henry00IS** [comment]
Personally, seeing people star or fork my own repositories was the highlight of my day. It gives me great joy and makes me want to continue working on my projects. But now it's just cluttered with everyone else's activity. It's why I like the old feed link people found:
https://github.com/dashboard-feed

![image](https://github.com/community/community/assets/7905726/352c8914-ebc5-4110-9889-bab26f879e7f)

It makes me happy and that's why I refresh that page all day.

**Reply by cyring** [reply]
> It's why I like the old feed link people found:
https://github.com/dashboard-feed

Bravo !

**Reply by esimov** [reply]
How does it looks like on a 27" monitor?

**Reply by wangrongding** [reply]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Reply by esimov** [reply]
I don't want to rely on hacky ways of installing 3rd party libraries or extensions to restore back something as elementary as showing who appreciated your library or who is following you. Github should fix this crap.

**Comment by port19x** [comment]
I want to see what repos my friends are starring and who is following me.
I choose to follow people for a reason, why are you taking this away?

**Reply by port19x** [reply]
Okay, so apparently getting who starred what works, but now the cards are fat af and include people I don't follow

**Comment by edermi** [comment]
The new feed contains less than 20 entries and I don't see how to go to page 2, 3, 4 etc.
Is this "improvement" an April's fools joke?

**Comment by molszanski** [comment]
Sorry, but it is really bad now

**Comment by gusmanb** [comment]
It's funny that they added a banner at the top that says "There’s improved filtering so you can customize your feed exactly how you like it". We liked it as it was, the new feed is useless and the filters make it even worse.

Bring back the old feed.

**Comment by fifsky** [comment]
My habits were broken and I couldn't find the knowledge I needed

**Comment by Hakky54** [comment]
Really sad to see this happen, I hope GitHub will listen to the feedback of their users. I think they might consider another platform like GitLab or something else to build up a community if more features like these are being enforced to the end-user...

**Comment by Havret** [comment]
It's completely useless. It seems they are trying to make another FB/Twitter/LinkedIn with GitHub. Terrible decision.

**Comment by gozeloglu** [comment]
I want to see the updates in chronological order. I don't want to see the "**X started following you**" update from last month on top of my feed. It is useless. Every morning I checked the latest updates(new followers, other people's starred repos, forks, etc.) on the feed. Now, it is totally mess. The previous version was working perfectly and it was clean. Please bring the previous feed back!

**Comment by pouriap** [comment]
The filter is a joke. I just filtered to only see stars and follows and it's showing me a blank page. Probably only shows events from the past few days. 

This is such an unprofessional and despicable move. Welcome to GitTok.

**Comment by alex-r-redfern** [comment]
How does GitHub keep making these awful UX changes? I can only assume this is being pushed down from a leadership team as no-one who uses the service regularly would think this is an improvement.

**Comment by jianxingxuejian** [comment]
Can't stand the new changes, I wonder which users need this kind of cluttered homepage that isn't even sorted by time

**Comment by morgoth** [comment]
Can you maybe add a filter option that would bring back the content from the old "Following" tab?
It's totally useless in current shape.

**Comment by eleev** [comment]
What a disaster this new feed is. I need the old version back! 🤬😡😖

**Comment by Alhadis** [comment]
Dear Microsoft:

<h1><em>PLEASE STOP TRYING TO TURN GITHUB INTO SOCIAL MEDIA.</em></h1>

Thanks. ❤️
— Another angry asshole

**Comment by Jak2k** [comment]
Let's make [our own fronted](https://github.com/Jak2k/Alternative-GitHub-UI/discussions) for GitHub to be more independent from it's UI changes.

**Comment by chuongmep** [comment]
Oh my got , not only me feel so bad with new feature, please revert feature.

**Comment by traviswimer** [comment]
So the feed no longer shows you information about yourself or your repositories? What exactly is the goal of this change, because it seems to be the exact opposite of what I want from the feed.

**Reply by linuxmobile** [reply]
Idk! This seems like it was the change for people who don't have any UX experience. I don't think there is a user happy with this change. That speaks to the importance of the company behind the change.

**Reply by traviswimer** [reply]
Yeah, although I think the UX for repo features is usually good, the UX for account level stuff tends to be unpleasant.

This change seems like it was probably intended to increase some useless metric like time spent on the site, which doesn't benefit the user or even their profits.

The new feed isn't necessarily bad overall. It looks like it does add some things for keeping up to date with repo updates/announcements, but I'm not sure why that requires removing other useful info from the feed.

**Reply by HCLarsen** [reply]
It seems to be the exact opposite of what everyone wants from the feed.

**Reply by linuxmobile** [reply]
> The new feed isn't necessarily bad overall.

I like the idea. But it's so poorly implemented that it ends up sucking. If I look from [OldFeed](https://github.com/dashboard-feed), I can see updates from a few minutes ago, from people I follow, companies, pull request. But if I go to the current feed, it shows me last followers from 12hs ago. The algorithm is really wrong.

**Comment by arpadgabor** [comment]
![image](https://github.com/community/community/assets/9270494/817cf873-a69f-4bcd-b968-5fcbd0af4204)

I was barely able to tell these 2 releases are actually from separate repositories...

**Comment by JPeisach** [comment]
On able top this, I star a ton of repositories, I've probably starred maybe 1000 total its either see every little PR/issue you don't care about or see nothing

**Comment by seefun** [comment]
+1 
There is a BUG! 
No matter how I choose the filter, I won't be able to find the star information of People I follow.
![image](https://github.com/community/community/assets/19152032/76ff437a-ee8e-4330-b8e1-8797c5388337)

I am completely unable to obtain any useful information on the github homepage now！

Terrible user experience

**Reply by esimov** [reply]
I would say that the whole Github UX is a giant bug.

**Reply by seefun** [reply]
> I would say that the whole Github UX is a giant bug.

You are right

**Comment by magicds** [comment]
old feed, please

**Comment by linuxmobile** [comment]
### Not even a response from the GitHub team.  What a mess

**Reply by takeda** [reply]
Why would they respond? They know users hate it, they are just hoping we will accept and move on like it was with twitter and reddit.

**Reply by linuxmobile** [reply]
You're right

**Comment by d2phap** [comment]
GitHub's next roadmap:
# "Introduce Bing News and Weather in Dashboard"

🤮

**Comment by virusdefender** [comment]
the new feed is garbage

**Reply by esimov** [reply]
If it's a garbage then it means it needs to be dropped to the place where it belongs to...to the trash bin. I'm totally upset.

**Comment by linuxmobile** [comment]
![image](https://github.com/community/community/assets/10554636/cb520d69-a125-430d-ae81-bd19a138568e)

I can't believe this shit.

**Comment by s60sc** [comment]
As the great philosophers said:
Quo magis res mutantur, eo magis in stercore vertuntur

**Comment by priverop** [comment]
With the new feed I can't find what I wanted from before: commits of the projects I'm following, PRs, etc

**Comment by paololazzari** [comment]
Hello github team, please run the following:

```bash
git reset --hard GOOD_FEED
git push -f origin master
```

**Comment by siddhpant** [comment]
Why did they ***again*** do this?

Last time they did this exact same thing, they ended up rolling it back. Why no lessons learnt?

**Comment by jenno-verdonck** [comment]
The new one is really useless. Bring back the old one pls.

**Comment by tuupola** [comment]
FFS. Stop trying to make GitHub Facebook. The new feed is useless. The feed is supposed to show what is happening in the repos I am following. Now it is just social media bs.

**Comment by jbjorge** [comment]
Feature request: I want to know what's happening in my repos and org-repos (and let me turn off everything else).

**Comment by tntyordan** [comment]
26703

**Comment by starcraft66** [comment]
This new feed sucks, it's full of spam

**Comment by JesseTG** [comment]
I want my chronological feed back. If I want to discover new repos, I'll look for them myself.

**Comment by CmST0us** [comment]
附议

**Comment by apertureless** [comment]
The new feed is horrible. 
The best thing in the feed was to see what people I follow do. What new repos they create, what repos they start. It was awesome to find new tools and software.

Now if I filter out "Recommended" and "Releases" . I see nothing! My feed is empty!

**Reply by seefun** [reply]
+1

**Comment by aoxiangtianji** [comment]
New feed is a piece of shit.

**Comment by real-re** [comment]
We need to check the feed for useful information, not this shit.

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._

## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 

_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_

- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._

_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Reply by smxi** [reply]
First class developer experience is a bit ambitious for github, I think if you aimed at a mediocre one you might achieve your goals.

Feed is still totally and utterly useless for any actual developer needs, a waste of screen space. Just switch back the old one and get rid of the people who pushed for this ridiculous idea. 

Don't shove changes on users, give users changes they request, this is so basic it's hard to understand what you guys are thinking.

I guess github want to start their move towards jumping the shark and making microsoft's investment worthless, this is a good first major step towards that laudable goal, similar to Musk buying twitter to help rid the world of that product.

The old feed was more or less ok, and gave developers roughly what they needed, seeing mainly activity on their projects, and on projects they follow or are interested in.

**Reply by linuxmobile** [reply]
> _With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._

How, as a user with over 8 years of participation on GitHub, I don't take changes like these lightly. However, I can't see at all how this change they've made, without even considering the users, could help me grow or, in any case, benefit GitHub users. It feels like a truly drastic measure, one in which nobody was taken into account except for your own intentions. It's painful that overnight, such a detrimental change was made.

I understand that you don't take changes lightly, but you also haven't consulted or listened to users **BEFORE** making this change. With the 'For You' tab, many of us were already unhappy. What leads you to make the decision to commit this atrocity?

> _We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

I'm glad that you're 'taking into account' the users, and that you've also responded. However, I would like to see reflected precisely the fact that you want to convey where this change is headed. I don't see any benefit for myself, and given the backlash this change has had, I also don't see a positive effect on others.

I would like to know, so I can stay informed, why you're making this decision and what steps you're going to take with GitHub? Previously, it was part of my routine to log in and see my feed. This allowed me to stay informed about what the people who inspire me to grow were doing. Whom they were following, which repositories they liked. Currently, I log in and feel an immense disgust that fills me with anger. You've truly turned my favorite platform into a headache.

## Despite the "changes" you decided to make. I still can't see my Feed. I don't understand what the changes are, despite seeing the changelog. As of today, I am still logging in and seeing NOTHING.

![image](https://github.com/community/community/assets/10554636/4e638dfa-9f08-429e-8540-cdf2c98a85cb)

**Reply by esimov** [reply]
> We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star.

Why you are just not showing the users who gives you a star or started following you? This is the only thing I need, the rest are just useless ads on which I'm completely uninterested and I don't opted for. I'm not following any organization and I don't want my feed to be populated with unrelated projects or any other unwanted repo suggestions.

**Comment by unitycoder** [comment]
another disaster UI/UX, completely useless.. +1 for old version back.

**Comment by Alhadis** [comment]
@linuxmobile Off-topic, but how did you get [that slick translucency](https://user-images.githubusercontent.com/10554636/267481093-4e638dfa-9f08-429e-8540-cdf2c98a85cb.png) in your browser?? (I have to ask, sorry).

**Reply by linuxmobile** [reply]
I'm using hyprland <3

**Reply by Alhadis** [reply]
Holy _hell_ that's a sexy-looking project. No idea why it's taken me this long to discover it. 😰 Thanks! 😀

**Reply by linuxmobile** [reply]
This is the time Alhadis <3

**Reply by ericoporto** [reply]
here is a link https://hyprland.org/

**Comment by lisenet** [comment]
I can't find what I'm looking for using the new feed. Need the old feed back.

**Comment by xieyuheng** [comment]
PLEASE STOP COMPLAINING !!!!!!!!!!!!!!!!!!!!

YOU VOICE MEANS NOTHING TO MICROSOFT !!!!!!!!!!!!!

**Reply by unitycoder** [reply]
reminds me of the Unity disaster (and the management not listening), same could easily happen with github..
imagine suddenly added fees for pushes (even if its minimal)

**Reply by xieyuheng** [reply]
Let's protest！！！！！

WE NEED THE OLD FEED BACK！！！

**Reply by unitycoder** [reply]
well spamming wont help, they dont read regular messages either : )

**Comment by h1994st** [comment]
Please take the old feed back

**Reply by xieyuheng** [reply]
WE NEED THE OLD FEED BACK！！！

**Comment by xieyuheng** [comment]
Please take the old feed back

**Comment by xieyuheng** [comment]
WE NEED THE OLD FEED BACK！！！

**Comment by smxi** [comment]
I got my old feed back, but this issue simply highlights the predictable issues when Microsoft bought github with no clear path towards future monetization. Obviously the mass scale copyright infringement which is somewhat cynically called 'copilot' was the first major salvo.
 
The tendency to hand one website domination of any area in tech is extremely unfortunate to put it mildly, and obviously when a corporation owns all the switches to control activity, and that corporation is required, at least in the USA, to maximize shareholder profit, there is simply no reason to believe that a site like github has any future with free software.

https://sfconservancy.org/GiveUpGitHub/  

That is a well thought out, carefully crafted, and typically high quality argument from the software freedom conservancy.

While I got my feed back to its normal state for some reason, this is simply an example of what happens when finance and marketing people get involved in technical engineering decision making processes. And this won't be the last time this happens on github.

I was particularly surprised to learn how hostile the original leader of github was to copyleft, aka, GPL, type software, which I find simply astonishing given how github was built on free software, probably top to bottom of the stack, but that's what dollar signs do to the human brain.

So the people who say, stop whining, those are totally right, if you have a free software project, then consider, as I am, moving to [codeberg.org](https://codeberg.org), or one of a handful of other non-corporate run code repos, sourcehut is mentioned but I don't know anything about them.

The record for corporate run code repos is quite dismal: googlecode, which was excellent, shut down, then closed. Github license violations en masse via GPT and copilot, then this little bizarre and very confused attempt to turn an engineering resource into some type of stupid social media platform (something so confused I would not for a second suggest that idea originated in the engineering section). Gitlab offers the same exact issues, it's a for profit corporation.

I would remind how easy it is to terminate a web resource, for example, the freenode IRC channel was acquired by some individual who then promptly attempted to start monetizing it, and quite literally almost overnight everyone had left for libera.chat, it was astonishing to see how quickly that happened. 

If you aren't doing free software, then a non free corporate repo is totally fine, no issues, it's like with like, but I am finding it increasingly odd to find any enthusiasm for hosting free software on such a site. Yes, the size of the developer base is useful, but objectively, how many so called _forks_ which are actually just a silly renaming of a clone actually ever have any activity? 

My notion is to leave  stubs for my repos here, then migrate most if not all to codeberg, and call it a day. This will be a bit tedious, but will also free me from thinking about what some executive at Microsoft comes up with as their next 'great idea to monetize the platform'.

I am heartened by Musk's ongoing successful attempt to destroy the twitter/X brand, that's going quite well, apparently the real valuation right now is about 1/10th the purchase price his group put out, but he's still having to pay about 1 billion a  year on the loans taken. So it's not really that hard to break apart an online resource if you really want to do it. Look at myspace for historical references, and in this context, look at sourceforge, googlecode, to realize how fragile this source repo thing is long term.

I've always thought it was somewhat absurd to rely on totally non free proprietary source repos for free software, that seemed like a major gap in the ecosystem, but there were no great alternatives in the past that you could count  on to be there in the future. But at some point, it's worth just saying, fine, I don't need this corporate stuff, and will go with the free as in software variant and just move on.

**Reply by gozeloglu** [reply]
how did you get old feed back?

**Reply by Hakky54** [reply]
Hi Gökhan, you can use https://github.com/dashboard-feed to view your old feed

**Reply by smxi** [reply]
They turned it back on, github is a black box, I did not do anything. That's why I have successfully now migrated all my repos to codeberg.org, and will over the next few months start shutting down and removing my code from github. For me this was the last straw, it just proved that github is now being run by non engineers. The real major issue I had was with the copyright violations of codepilot, which maybe a big enough legal team from microsoft will get legally accepted, but in the real world, not the legal one, it's totally obvious what they did and are doing with the code in their repos, this may be the biggest single mega-corporate copyright violation since google books decided it was ok to make copies of copyrighted works from libraries.

I am fortunate in that I have always loathed github and it's entire way of functioning, so for me leaving was more of a relief than anything else, that won't be the same for other users, but for me, it was an easy decision to make.

**Reply by tuupola** [reply]
There are several userscripts to get the old feed back. I am using this:

https://github.com/SamJakob/github-homepage

**Comment by robmarkcole** [comment]
Plus one - given the new feed a try since launch but honestly it is a downgrade and I now get no value from it

**Comment by dennis1248** [comment]
Also I am sure 10 other people have already said;

I would highly prefer to keep what I actively follow and stuff Github recommend seperate, I have completely lost my overview on the stuff I actually care about since the stuff I do care about is not hugely active and thus drowned out by the recommendations.

**Comment by nihongeakira** [comment]
**I feel the Same way I still Remember** _Those_ day Here ### Image [https://webergrillsparts.com/genesis-silver-parts-a-b-c/](url)

**Comment by unitycoder** [comment]
@github I've clicked " want to see fewer ..." probably hundred of times, how long does it take for that AI to get it?
You could replace the expensive AI system with literally 10 lines of "if user wants to see these then show it else dont"

**Reply by smxi** [reply]
AI is a marketing term, all it refers to currently is crude brute force pattern matching, made possible by using billions of dollars of hardware coupled with tons of bandwidth and ever increasing training sets. These are all the exact opposite of what intelligence is, and how it works. 

None of the big 3 (Google, Tesla, OpenAI) currently have anything even faintly resembling a working model of cognition, let alone intelligence, artificial or otherwise. All are following a crude, failed model of cognition that went out of fashion in the 1950s, but unfortunately, engineering students don't take serious non engineering classes so they just sort of glommed onto this old model because it's the only one they can figure out how to implement with computers and code. 

If they were to follow Musk's advice which he himself is totally ignoring and go to first principles, my guess is they'd have to dump all their hardware and tools and start over again, maybe they can salvage a bit here and there, but it's all built on the wrong foundation, and none of the big 3 are going to admit it because they dropped too much money into their attempts. 

Next generation, or the one after that, won't be so dumb though.

**Comment by cambiata** [comment]
After this change, Github isn't the source of knowledge and inspiration it used to be. 

Has there been any kind of public discussion where the people responsible for these changes has spoken out and responded to criticism in a serious way?

**Reply by smxi** [reply]
github is a division of the Microsoft corporation. Corporate divisions don't need to engage in public discussions about changes, though ideally they  would have the brains to at least do studies and get feedback to make sure they aren't making big mistakes, but when it comes to maximizing capital and cash flows from a large purchase, it's about money, nothing else. Microsoft in USA as a publicly traded corporation is actually required legally to maximize shareholder value. 

This is why governments or other non corporate actors are foolish to rely on corporate media like twitter, or corporate git repo hosting like github.

They have no obligation to first of all admit who was responsible for this last action, because that was obviously not engineers, it was some type of finance or marketing type group who is totally unclear about what a source code repository is, what its function is, and why paying 4 billion for one  was a really stupid move in the first place because git by its nature is decentralized, so they are relying on value added features like copyright inifringement via copilot and other stuff to lock people in and ideally start generating rentier type monthly cash flows for various features.

Moving away from such corporate entities and ownership is the solution, same as when Linus Thorvalids first wrote git when the author of bitkeeper decided to make it's use as non free software critical to the survival of the Linux project.  Linus has for years been criticized for having the bitkeeper non free software be a single point of failure for the Linux project, and when that concern finally became a reality, he wrote git, but the real mistake was in the initial reliance on non free software to do a core role in the Linux project. The same issue exists today with free/open source projects relying on github for antyhing.

The more people and projects move away, the greater resources are then available for generating longer lasting alternate solutions to this issue, but it has to start somewhere, and complaining about a corporation doing what corporations always do is probably not going to do anyone much good.

**Reply by quantuminformation** [reply]
good one

**Comment by bluemix** [comment]
No response from GitHub for about a month. I think they want us to adapt forcibly.

**Comment by PandaXR** [comment]
Looks like they just decided to ignore the problem. GitHub, the new feed is useless. Bring back the previous feed.

**Comment by brunomikoski** [comment]
Please, pretty please <3

**Comment by alexandercerutti** [comment]
I mean, after over a month, I got used to the new dashboard and I feel it to be almost (**ALMOST**) as it was before.

As I care of stars on my projects (yay, dopamine!), after a few feedbacks on the previous threads, I started see them again (except those from private profiles, which seems to be coherent with the old feed).

What I really do not stand is the new ambiguity between "starring" and "watching".
I can accept receiving suggestions in the middle (yeah, I usually ignore them), but **I cannot stand receiving updated about a repository I'm not watching**.

Both Announcements and Releases. This ambiguity completely defeats the purpose of watching a repository because interested in it.

I do have some repos I'm watching because I want to keep updated, but I also star many repositories to save them and to support them, but I'm really not interested.

Removing such ambiguity, for me, would be the next best thing to do to improve the new feed.

**Comment by janbrasna** [comment]
Thanks GitHub, this dumbed–down dashboard now not displaying wiki updates/diffs from our repos means I've missed a crucial feature change in a core product lifecycle that I should've reviewed.

So this change effectively means actual harm being done to our orgs, compared to the old feed. Well done! (not) 🙈

**Comment by SheTieJun** [comment]
may need tab cards to record and display the filter items I choose. I want to directly see what I want to see based on my wishes.

**Comment by molvqingtai** [comment]
I need to pay attention to people's dynamics, not the current automatic recommendation.

**Comment by owen800q** [comment]
Why missing show more button now? https://github.com/dashboard-feed?

<img width="1018" alt="Screenshot 2023-10-21 at 11 31 40 PM" src="https://github.com/community/community/assets/19252045/ec51b6f3-c572-416c-8bd3-0db212d4e94e">


Original

<img width="1018" alt="Screenshot 2023-10-21 at 11 32 01 PM" src="https://github.com/community/community/assets/19252045/3da10ecb-ac39-4ab1-a7a0-738a4119cc4f">

**Comment by mokeyish** [comment]
We need old feed back

**Comment by dsyx** [comment]
I like the old one, it's simpler and clearer. I still find the new one confusing me.

Perhaps there should be a feed-style toggle on Feature Preview.

Thanks

**Comment by linuxmobile** [comment]
### WE NEED THE OLD FEED BACK AGAIN.

**Please, maintain this Discussion active.**

**Comment by molszanski** [comment]
This new feed is a disaster. I've unstarred some repositories because the feed couldn't learn that I don't want to see news about every new discussion in some low-prio-for-me repo. 

It is insane how bad the new one is.

**Reply by molszanski** [reply]
Now I don't look at this feed at all. It is just trash info

**Comment by laibe** [comment]
I agree, the feed has gotten way worse. Sometimes it returns something with filters enabled, a few minutes later it does not, then again, it has been really unreliable. Please bring back the old feed!

**Comment by molszanski** [comment]
I found a solution that works for me. I am not using feed anymore altogether. 
Any pageviews there are accidental

**Reply by danielhoherd** [reply]
Same for me. I'm so glad that <https://github.com/dashboard-feed> still works.
